db/migrate/20150410124111_add_nic_provider_config_to_compute_site.rb

```diff
@@ -0,0 +1,5 @@
+class AddNicProviderConfigToComputeSite < ActiveRecord::Migration
+  def change
+    add_column :atmosphere_compute_sites, :nic_provider_config, :text
+  end
+end
```

Comment:
We've had a discussion with @mkasztelnik  about it and decided not to change it.